.envrc

```diff
@@ -88,7 +88,9 @@ export PATH
 export FISSILE_CONFIG_PREFIX=scf
 
 # We will persist compilation results to the VM here
-export SCF_PACKAGE_COMPILATION_CACHE="${SCF_PACKAGE_COMPILATION_CACHE:-${PWD}/.fissile/compilation}"
+if [ -z ${SCF_PACKAGE_COMPILATION_CACHE+x} ]; then
```

Comment:
What is the `+x` at the end of the variable name?

Reply:
Returns true if the var is unset, false if it's a string of length >= 0

Reply:
Ok, thanks!

Maybe add a comment above that explains that you want to set the default for an unset variable, but want to keep the empty string because that disables caching?

However, I think I would prefer to use a separate variable `SCF_DISABLE_PACKAGE_COMPILATION_CACHE` instead of making the empty string a magic value distinct from the null value.